sdk/test-utils/recorder/GUIDELINES.md

```diff
@@ -8,6 +8,12 @@
   - Tests hit the live-service
   - [Nock](https://www.npmjs.com/package/nock)/[Nise](https://www.npmjs.com/package/nise) are leveraged for recording the request-responses for future use
   - If recordings are already present, forces re-recording
+  - The recorded files contain a hash of the test that executed that recording.
+  - Some recorded files might not have the hash if they were generated by an older version of the recorder.
+- If TEST_MODE = "soft-record",
+  - Tests will be skipped if they haven't changed.
```

Comment:
```suggestion
  - Tests will be skipped if they haven't changed. `soft-record` mode is isomorphous to `record` mode in all other aspects.
```

??
Just mention that the soft-record mode is the same as the record mode everywhere else in the document.